api/src/main/java/org/openmrs/module/atomfeed/AtomfeedmoduleConstants.java

```diff
@@ -0,0 +1,8 @@
+package org.openmrs.module.atomfeed;
+
+public final class AtomfeedmoduleConstants {
+	
+	public static final String ATOMFEED_PATH_TO_DAFAULT_CONFIGURATION = "default.json";
```

Comment:
Change filename to defaultAtomfeedConfig

Reply:
Corrected.